src/indra_cogex/sources/goa/__init__.py

```diff
@@ -42,10 +42,14 @@ def __init__(self):
         self.df = load_goa(GOA_URL)
 
     def get_nodes(self):  # noqa:D102
-        for go_node in self.df["GO_ID"].unique():
-            yield Node("GO", go_node, ["BioEntity"])
+        for go_id in self.df["GO_ID"].unique():
+            yield Node.standardized(
+                db_ns="GO", db_id=go_id, labels=["BioEntity"]
+            )
```

Comment:
Doing standardization here could have unintended consequences: it can make db_ns/db_id be mapped to some other namespace which for the purposes of GO enrichment analysis will be a problem.

Reply:
Resolved in https://github.com/gyorilab/indra_cogex/pull/149/commits/7b2e1c540bda06991c10728bd893afcafb7ca407